packages/react/CHANGELOG.md

```diff
@@ -1,8 +1,5 @@
-### 2.2.0
-- Sets the default `apiVersion` to `v1`.
-
-### 2.1.0
```

Comment:
I don't think it makes sense to hide old information. It's better to keep it in case someone's using that version, so they aren't confused about what's going on